torch/csrc/jit/tensorexpr/ir.cpp

```diff
@@ -83,7 +83,8 @@ StorePtr BufHandle::store(
 
 ExprPtr flatten_index(
     const std::vector<ExprPtr>& dims,
-    const std::vector<ExprPtr>& indices) {
+    const std::vector<ExprPtr>& indices,
+    const std::vector<ExprPtr>& strides) {
```

Comment:
We don't need `dims` anymore I think?

Reply:
Yes, if strides are not optional here - we can remove dims.